src/test/resources/com/jpinpoint/perf/lang/java/ruleset/common/xml/AvoidLombokAnnotationForNonExistentFields.xml

```diff
@@ -38,10 +37,9 @@ import lombok.Setter;
     <test-code>
         <description>violation: Lombok annotation for fields used while class has no fields</description>
         <expected-problems>5</expected-problems>
-        <expected-linenumbers>4,5,12,19,34</expected-linenumbers>
+        <expected-linenumbers>3,4,11,18,33</expected-linenumbers>
         <code><![CDATA[
-import lombok.Getter;
-import lombok.Setter;
+import lombok.*;
 
 @Getter // bad
 @Setter // bad
```

Comment:
Weird to have these annotations on Exception anyway?

Wondering: what if there is a class extends and the (only) super class has fields? Might that be valid?

Reply:
By my knowledge the annotation should then be at the super class level

Reply:
indeed weird